services/app-api/prisma/schema.prisma

```diff
@@ -20,15 +20,15 @@ model TestUser {
 
 // The StateSubmission Form Data
 model StateSubmission {
-  id                  String                    @id
-  state               State                     @relation(fields: [stateCode], references: [stateCode])
-  stateCode           String
-  revisions           StateSubmissionRevision[]
```

Comment:
my autosave did this

Reply:
I gotta figure out all the linters. I hope this is the result of `prisma format`